apps/src/blockly/addons/contextMenu.js

```diff
@@ -2,7 +2,6 @@ import GoogleBlockly from 'blockly/core';
 import msg from '@cdo/locale';
 
 import {Themes, MenuOptionStates, BLOCKLY_THEME} from '../constants.js';
-
 const registerDeletable = function() {
   const deletableOption = {
     displayText: function(scope) {
```

Comment:
Mind fixing typo in line 8?

Reply:
What's the typo?

Reply:
isDeletale

Reply:
Is that not a typo??

Reply:
Oh, I see it now. It's in the line below this thread so I missed it.